test-tests/shared/src/test/scala/zio/test/environment/RandomSpec.scala

```diff
@@ -1,49 +1,120 @@
 package zio.test.environment
 
+import zio._
+import zio.ZIO._
```

Comment:
I wouldn't do this import. It works okay here but in other cases can create issues because similar methods exist on, for example, `ZIO`, `ZManaged`, and `ZStream` so it can make it less clear what is being done.